2024Robot_Team3484/src/main/cpp/subsystems/IntakeSubsystem.cpp

```diff
@@ -0,0 +1,92 @@
+//
+// Intake Subsystem
+//
+// -Noah, Aidan  & Ethan
+//
+
+#include <subsystems/IntakeSubsystem.h>
+#include <units/angle.h>
+
+using namespace IntakeConstants;
+
+IntakeSubsystem::IntakeSubsystem() {
```

Comment:
Pass CAN ID values through the constructor. This is more of proper programming practice in general instead of referencing the constants in the header file.
```
IntakeSubsystem::IntakeSubsystem(int ID_Pivot, int ID_Drive) : 
    _pivot_motor(ID_Pivot), _drive_motor(ID_Drive)
{

}
```